strings/ion_laws.json

```diff
@@ -291,6 +291,7 @@
         "MERCURY",
         "JUPITER",
         "URANUS",
```

Comment:
```suggestion
```

Reply:
I wanted to keep both, so they're both options.